core/controllers/question_editor.py

```diff
@@ -29,14 +29,29 @@
 class QuestionCreationHandler(base.BaseHandler):
     """A handler that creates the question model given a question dict."""
 
+    def validate_skill_ids(self, skill_ids):
+        """Validates that the passed skill ids are a list of strings."""
+        if not(isinstance(skill_ids, basestring)) and not(
+                all([isinstance(
+                    skill_id, basestring) for skill_id in skill_ids])):
+            raise self.InvalidInputException
+
     @acl_decorators.can_manage_question_skill_status
-    def post(self, skill_id):
+    def post(self, comma_separated_skill_ids):
         """Handles POST requests."""
-        skill_domain.Skill.require_valid_skill_id(skill_id)
-        skill = skill_services.get_skill_by_id(skill_id, strict=False)
-        if skill is None:
+        try:
+            skill_ids = comma_separated_skill_ids.split(',')
+            self.validate_skill_ids(skill_ids)
+            for skill_id in skill_ids:
+                skill_domain.Skill.require_valid_skill_id(skill_id)
+        except Exception:
+            raise self.PageNotFoundException
+
+        try:
+            skill_services.get_multi_skills(skill_ids)
```

Comment:
That function currently raises an exception: https://github.com/oppia/oppia/blob/46ae61e59d21952014d925fb7cb966bbc93ee9c2/core/domain/skill_services.py#L199

I will pass on the exception text as well so that we know which skill ID is missing.

Reply:
Done